Source/OxyPlot/Series/BarSeries/ErrorBarSeries.cs

```diff
@@ -53,7 +53,7 @@ protected internal override void UpdateMaxMin()
             base.UpdateMaxMin();
 
             //// Todo: refactor (lots of duplicate code here)
```

Comment:
Is this still needed?

Reply:
I would argue yes. The `UpdateMaxMin()` implementation of `ErrorBarSeries` is quite identical to the implementation of `BarSeries`, which I think this comment refers to.